sku.0/sys.server/compiled/game/script/theme_park/wod/wod_mutant_rancor_combat.java

```diff
@@ -0,0 +1,229 @@
+package script.theme_park.wod;
+
+import script.*;
+import script.base_class.*;
+import script.combat_engine.*;
+import java.util.Arrays;
+import java.util.Hashtable;
+import java.util.Vector;
+import script.base_script;
+
+import script.library.ai_lib;
+import script.library.beast_lib;
+import script.library.buff;
+import script.library.chat;
+import script.library.combat;
+import script.library.create;
+import script.library.group;
+import script.library.prose;
+import script.library.static_item;
+import script.library.trial;
+import script.library.utils;
+
+public class wod_mutant_rancor_combat extends script.base_script
+{
+    public wod_mutant_rancor_combat()
+    {
+    }
+    public static final String NS_BICEP_SCHEMATIC = "item_wod_bossloot_ns_bracer";
+    public static final String SMC_BICEP_SCHEMATIC = "item_wod_bossloot_smc_bicep_";
+    public static final String SMC_BRACER_SCHEMATIC = "item_wod_bossloot_smc_bracer_";
+    public static final int MAX_DISTANCE = 104;
+    public static final int REPEAT_TIME = 45;
+    public static final int OFFSET_TIME = 15;
+    public static final int HP_WOD_RANCOR = 618125;
```

Comment:
I can't find any documentation suggesting this health value over the default HP for a level 95 Boss. Not sure where this value came from?